src/cli_plugin/install/zip.js

```diff
@@ -0,0 +1,72 @@
+import _ from 'lodash';
+import DecompressZip from '@bigfunger/decompress-zip';
+
+//***********************************************
+//Creates a filter function to be consumed by extractFiles
```

Comment:
A general note that applies to the whole PR: I love that you're adding comments for functions, but in future PRs it'd be great if we could standardize around the use of [jsdoc](http://usejsdoc.org/) comment styling.